qcodes/tests/test_visa.py

```diff
@@ -134,7 +134,14 @@ def open_resource(self, address):
         self.assertEqual(rm_mock.call_args, ((),))
         self.assertEqual(address_opened[0], 'ASRL2')
 
+        # this one raises a warning
```

Comment:
If it raises a warning we should catch that and perhaps also test that it indeed does raise

Reply:
That is a good idea. I have added that now.